ndcube/utils/cube.py

```diff
@@ -195,3 +196,59 @@ def _get_dimension_for_pixel(axis_length, edges):
         False stands for pixel_value, while True stands for pixel_edge
     """
     return axis_length+1 if edges else axis_length
+
+def _get_extra_coord_edges(value, axis=-1):
```

Comment:
Maybe the name of the function can be changed to some appropriate one :P

Reply:
If you are looking for suggested name change how about ```_infer_linear_extra_coord_edges```?